net-mail/email/files/email-3.1.3-fix-clang16-configure.patch

```diff
@@ -0,0 +1,19 @@
+Clang16 will not allow using undeclared library functions by default.
+This imports ctype.h into that check so isdigit() can be used without errors.
+
+Bug: https://bugs.gentoo.org/879737
+
+PR upstream: https://github.com/deanproxy/eMail/pull/63
```

Comment:
❤️

Reply:
:bow: 

I don't even know if that is the correct upstream, we are not getting the SRC_URI from there....